data_pipelines_cli/cli_commands/dbt.py

```diff
@@ -1,15 +1,34 @@
-import subprocess
 from typing import Tuple
 
 import click
 
+from ..cli_constants import profiles_build_path
+from ..cli_utils import echo_info, echo_subinfo, subprocess_run
+from ..config_generation import copy_config_dir_to_build_dir, generate_profiles_yml
 
-def dbt(command: Tuple[str, ...], target: str) -> None:
-    subprocess.run(["dbt", *command, "--profiles-dir", ".", "--target", target])
+
+def dbt(command: Tuple[str, ...], env: str, generate_profiles: bool = True) -> None:
```

Comment:
I've decided to keep `dp dbt` anyways, as one could want to call a `dbt` command different from `run` or `test`. However, if using it got too unintuitive, we can safely remove `dp dbt` support.

Reply:
I think now we should skip dbt. Is someone wants to use other command he can always use dbt directly.

Reply:
`dp dbt ...` removed